app/views/shared/export/_plan_txt.erb

```diff
@@ -30,7 +30,7 @@
     <% section[:questions].each do |question| %>
       <% if @show_sections_questions %>
         <%# text in this case is an array to accomodate for option_based %>
-        <% if question[:text].length > 1 %>
+        <% if question[:text].is_a?(Array) %>
```

Comment:
Can you change it to:
```
question[:text].respond_to(:each) 
```
using duck typing is safer because in the case that hash key had an ActiveRecord::Relation, that is not an object of Array class however it responds to each method to iterate the collection.